content/influxdb/cloud-dedicated/query-data/troubleshoot.md

```diff
@@ -0,0 +1,194 @@
+    - [Invalid Argument Error: bucket <BUCKET_ID> not found](#invalid-argument-error-bucket-bucket_id-not-found)
+      - [Unauthenticated: Unauthenticated](#unauthenticated-unauthenticated)
+    - [Unauthenticated: read:<BUCKET_ID> is unauthorized](#unauthenticated-readbucket_id-is-unauthorized)
+
+## InfluxDB Flight responses
+
+{{% cloud-name %}} provides an InfluxDB-specific Arrow Flight service that uses gRPC Remote Procedure Calls (gRPC) to transport query result data in Arrow format.
+Flight defines a set of [RPC methods](https://arrow.apache.org/docs/format/Flight.html#rpc-methods-and-request-patterns) for servers and clients.
+Using gRPC, a Flight client (for example, `influx3` or an InfluxDB v3 client library) calls an InfluxDB Flight RPC server method to send a request to the server.
+In gRPC, every call returns a status object that contains an integer code and a string message.
+While gRPC defines the integer [status codes](https://grpc.github.io/grpc/core/status_8h.html) and definitions for servers and clients,
+a server can choose which status to return for an RPC call.
+During a request, the gRPC client and server may each return a status independent of the other.
+
+InfluxDB v3 client library query methods implement the Flight [`DoGet(FlightCallOptions, Ticket)` method](https://arrow.apache.org/docs/cpp/api/flight.html#_CPPv4N5arrow6flight12FlightClient5DoGetERK17FlightCallOptionsRK6Ticket) that retrieves a single stream for the the request.
+For example, if you call the `influxdb3-python` Python client library `InfluxDBClient3.query()` method, the client in turn calls the `pyarrow.flight.FlightClient.do_get()` method and passes a Flight ticket with your credentials and query to InfluxDB.
+
+InfluxDB v3 can respond with one of the following:
+
+- The schema for record batches in the stream, followed by a stream of query result data (`RecordBatch`), the request status (`OK`), and optional trailing metadata
+- The schema, a stream or partial stream of data, and no status (for example, in the event of a network interruption)
+- An [error status code](#influxdb-error-codes) and an optional `details` field that contains the status and a message that describes the error
+
+_To learn more about Flight RPC requests, see the [Apache Arrow Flight RPC documentation](https://arrow.apache.org/docs/format/Flight.html)_
+
+When the Flight client receives a stream, it reads each `RecordBatch`  from the stream until there are no more messages to read, and considers the request complete when it has received all the messages.
+
+### Schema
+
+A Flight response can contain a schema that describes the size and columns in the query result data.
+You can inspect the schema in a {{% cloud-name %}} response to find the data type and InfluxDB column type (timestamp, tag, or field) for columns in the data set.
+Data transformations can use the schema when converting Arrow data to other formats and back to Arrow.
+
+The schema has the following structure:
```

Comment:
It might be good to simply refer to the spec here rather than trying to replicate some portion of it:

https://arrow.apache.org/docs/format/Columnar.html#format-ipc